### DISCUSSION TITLE
About the issue of the student package on GitHub.

### DISCUSSION THREAD

**Discussion by congyijiu** [community/community#65676] [GitHub Education]
### Hi everyone,

![1](https://github.com/community/community/assets/113180624/2be00a6a-ace7-45d9-8dea-3eb4dc02b3f9)
I cannot apply for my school email, but I can provide my student ID. I used to apply like this before, but this time it requires me to use the school email forcibly. Is there any way to solve this?
I successfully applied for the GitHub Student Pack in March this year, but recently it suddenly expired. What happened?

**Comment by arafajs** [comment]
1. Contact your school's IT department and ask them if they can provide you with a school-issued email address that you can use to apply for the GitHub Student Developer Pack.
2. If your school does not provide school-issued email addresses, you can try uploading documents that prove your current student status, such as a copy of your student ID, transcript, or enrollment verification letter.
3. You can also try contacting GitHub support and explaining your situation. They may be able to make an exception and approve your application without a school-issued email address.
